### ISSUE TITLE
Attach File named URL

### ISSUE THREAD

**Issue by Zerick** [frappe/erpnext#37377]
When using the Attach File feature to link URLs in ERPNext, I would like to be able to name the links rather than view an abbreviated form of the URL in the Attachments list.

See attached

![ERPNext Feature request for URLs](https://github.com/frappe/erpnext/assets/1748040/25db5412-d6e6-4226-8b43-b8e171296d69)

**Comment by bosue** [comment]
True. This should also be a regular customizable doctype with a regular customizable quick entry form (+ some extra features).
